app/views/kids/show.html.erb

```diff
@@ -24,15 +24,15 @@
     </div>
     <div class="white-card">
       <p>Amount of scanned tweets</p>
-      <%= line_chart Statistic.group_by_day(:created_at).count %>
+      <%= line_chart Statistic.group_by_day(:created_at).count, label: " scanned tweets", min:0, max:400, curve: false %>
```

Comment:
The label isn't shown it seems - But there is already a header saying Amount of scanned tweets so should be fine either way.